__tests__/app.test.js

```diff
@@ -38,54 +37,80 @@ describe("Endpoint: GET api/topics", () => {
+    .get("/api/articles")
+    .expect(200)
+    .then((result) => {
+      expect(result.body.articles).toHaveLength(12);
+    });
+  });
+  test("Returns an array of objects with correct properties", () => {
+    return request(app)
+    .get("/api/articles")
+    .expect(200)
+    .then((response) => {
+      response.body.articles.forEach((article) => {
+        expect(article).toMatchObject({
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+          comment_count: expect.any(String),
+          article_id: expect.any(Number)
+        })
       });
+    });
+  });
+  test("The article objects are sorted in descending order by their date property", () => {
+    return request(app)
+    .get("/api/articles")
+    .expect(200)
+    .then((response) => {
+      expect(response.body.articles).toBeSortedBy("created_at", {descending: true})
+    });
   });
 });
 
-describe("Endpoint: GET /api/articles", () => {
-  test("Returns a status code of 200 when query exists, and the correct length array", () => {
+describe("Endpoint: GET api/articles/:article_id", () => {
+  test("Returns a status code of 200 when query exists", () => {
     return request(app)
-      .get("/api/articles")
-      .expect(200)
-      .then((result) => {
-        expect(result.body.articles).toHaveLength(12);
-      });
+    .get("/api/articles/1")
+    .expect(200)
   });
-
-  test("Returns an array of objects with correct properties", () => {
+  
+  test("Returns a single article object with an id which matches the parametric id, and article has the correct properties", () => {
     return request(app)
-      .get("/api/articles")
-      .expect(200)
-      .then((response) => {
-        response.body.articles.forEach((article) => {
-          expect(article).toMatchObject({
-              title: expect.any(String),
-              topic: expect.any(String),
-              author: expect.any(String),
-              body: expect.any(String),
-              created_at: expect.any(String),
-              votes: expect.any(Number),
-              article_img_url: expect.any(String),
-              comment_count: expect.any(String),
-              article_id: expect.any(Number)
-            })
-        });
+    .get("/api/articles/1")
+    .expect(200)
+    .then((result) => {
+      console.log(result.body.article);
+      expect(result.body.article).toMatchObject({
+        article_id: expect.any(Number),
+        title: expect.any(String),
+        topic: expect.any(String),
+        author: expect.any(String),
+        body: expect.any(String),
+        created_at: expect.any(String),
+        votes: expect.any(Number),
+        article_img_url: expect.any(String),
       });
-  });
+    }
+    )
+  })
+});
 
```

Comment:
Error handling should be added here for this end point i.e. if an id is given which does not exist (9999), an invalid id is given (not-a-id)